src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Altinn.Platform.Authentication.Model;
+using Altinn.Platform.Authentication.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Platform.Authentication.Controllers
+{
+    /// <summary>
+    /// CRUD API for the System User 
+    /// </summary>
+    /// [Authorize]
+    [Route("authentication/api/v1/systemuser")]
```

Comment:
i think we can have something like  <<authentication/api/v1/systemusers>> here and remove the systemuser from rest of the routes